src/reusableState/customList/StringListReducers.ts

```diff
@@ -54,6 +54,14 @@ export const stringListCompositeReducer = (
                     list: [..._.without(state[action.payload.id].list, action.payload.value)],
                 },
             };
+        case StringListActions.updateValues:

```

Comment:
`StringListActions.add` is almost identical, I'm pretty sure you can combine them so that they use the same logic.

Reply:
Good point. I added this action to avoid confusion when you use the action in your dispatch. I can be confusion to use an add when your action its supposed to only update an existing object.

Reply:
Fixed